packages/cli/src/ui/hooks/slashCommandProcessor.ts

```diff
@@ -158,6 +158,7 @@ export const useSlashCommandProcessor = (
   );
   const commandContext = useMemo(
     (): CommandContext => ({
+      signal: new AbortController().signal, // Default signal, will be overridden in handleSlashCommand
```

Comment:
Hmm isa there still value in memoizing this since we just re-expand in `handleSlashCommand` this has a bit of code smell?